api/src/wfl/server.clj

```diff
@@ -85,11 +86,34 @@
      (setDebugEnabled [enabled])
      (warn ([thrown]) ([msg & args])))))
 
+(defn ^:private start-workload-manager []
+  (letfn [(update-workloads! [])] ;; todo
+    (pprint "starting workload update loop")
```

Comment:
What are the pros and cons of routing this through `pprint` rather than `log/info`?

Reply:
None - this module doesn't use any of the log stuff presumably because it interferes with jetty (hence tom's `stfu-jetty`)

Reply:
Though, having commented out `stfu-jetty`, i'm not sure what exactly it was doing before

Reply:
I could be totally off the mark here--I think `stfu-jetty` predated our usage of logging and was just another way to quiet it down. There's actually a small chance `stfu-jetty` isn't necessary anymore because [I might've managed to configure Log4j2 to not report anything less than errors from Jetty](https://github.com/broadinstitute/wfl/blob/main/api/resources/log4j2.xml#L21)

Reply:
Switched to `log/info` - jetty seems to have got the picture.